pom.xml

```diff
@@ -21,7 +21,7 @@
                     <testFailureIgnore>true</testFailureIgnore>
                     <perCoreThreadCount>false</perCoreThreadCount>
                     <includes>
-                        <include>**/CukesRunner*.java</include>
```

Comment:
Line 24 is showing a conflict because in this project we called our runner CucumberRunner so the pom.xml is not recognizing CukesRunner. I think if you rename "CukesRunner" to "CucumberRunner" then the conflict will resolve.